thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -454,8 +457,8 @@ public Response updateAnomalyFunction(@NotNull @QueryParam("id") Long id,
    * an activated anomaly detection function
    */
   @POST
-  @Path("/anomaly-function/apply/{autotune_config_id}")
-  public Response applyReplayConfig(@PathParam("autotune_config_id") @NotNull long id,
+  @Path("/anomaly-function/apply/{autotuneConfigId}")
+  public Response applyAutotuneConfig(@PathParam("autotuneConfigId") @NotNull long id,
       @QueryParam("cloneFunction") @DefaultValue("false") boolean isCloneFunction,
       @QueryParam("cloneAnomalies") Boolean isCloneAnomalies) {
```

Comment:
there is a logic inside this enpoint, if isClonedAnomalies is not given, should always clone if the original function has labeled anomalies. This is to protect user information.